src/Geocoder/Provider/GoogleMaps.php

```diff
@@ -153,8 +153,9 @@ private function executeQuery($query)
             throw new NoResult(sprintf('Could not execute query "%s".', $query));
         }
 
-        if ('REQUEST_DENIED' === $json->status && 'The provided API key is invalid.' === $json->error_message) {
-            throw new InvalidCredentials(sprintf('API key is invalid %s', $query));
+        if ('REQUEST_DENIED' === $json->status) {
```

Comment:
I don't really get the point of this change. Could you explain why you removed the second condition?